app/javascript/components/projects/ProjectView.jsx

```diff
@@ -87,25 +92,38 @@ class ProjectView extends React.Component {
         edit_button = <span></span>;
     }
 
+    let skillList = null;
+    
+    if (this.props.skills) {
+        skillList = this.props.skills.map((skill, index) => {
+            var skill_pill = (
+                <p>skill</p>
+            )
+        })
+        return (
+            <div>{skill_pill}</div>
```

Comment:
I think the return should be encapsulated in the `map`, since each mapping should return some element.